### PR TITLE
Add private preview use

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/issues/482

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -2470,6 +2470,8 @@ The valid values are:
 > a. MajorVersionLimit accepts values from 1 through 50,000 (inclusive).<br/>
 > b. ExpireVersionsAfterDays accepts values of 0 to Never Expire or values >= 30 to delete versions that exceed that time period.<br/><br/>
 > When Version History Limits are managed Automatically (EnableAutoExpirationVersionTrim $true), setting MajorVersionLimit or ExpireVersionsAfterDays will result in an error as the count limits are set by the service.
+>
+> This parameter is currently under private preview use.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2474]:
```suggestion
> This parameter is currently under private preview.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@get-itips

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Comment by yogkumgit** [comment]
@tseward

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**